packages/grid/x-data-grid/src/components/cell/GridActionsCell.tsx

```diff
@@ -211,6 +225,7 @@ function GridActionsCell(props: GridActionsCellProps) {
           open={open}
           target={buttonRef.current}
           position={position}
+          disablePortal
```

Comment:
Without it, the styles aren't applied to the elements because they're outside of grid DOM tree